omero/sysadmins/unix/server-centos6.txt

```diff
@@ -188,23 +189,14 @@ See also :doc:`install-web/install-nginx`.
 
 **The following steps are run as root.**
 
-To install Nginx run::
-	
-	yum install nginx
-
-In the virtual environment, install the requirements to run OMERO.web using the requirements file::
+Install Nginx, install the requirements to run OMERO.web in the virtual environment, deactivate it
+and copy the Nginx OMERO configuration file into the
+Nginx configuration directory, and disable the default configuration::
```

Comment:
You need to delete the second `:` here, it's making the literal include below be rendered as a code box rather than obeying the directive and putting the text from the script in the box.